package.json

```diff
@@ -181,7 +181,7 @@
     "publish-components": "npm run script scripts/publish-components/index.ts $1 -- $@",
     "script": "babel-node --extensions .js,.ts,.tsx -- $@",
     "start": "node dist/server",
-    "start:ci": "nyc --silent --exclude \".next/**\" node server",
+    "start:ci": "nyc --silent --exclude \".next/**\" node dist/server",
```

Comment:
How is that related to the problem? Can we look at this in isolation?

Reply:
Yep, it was a rabbit hole. I'm removing from this PR.